versions/application.txt

```diff
@@ -0,0 +1 @@
+1.6
```

Comment:
Rather than having a separate file with the version number, a better approach might be to have an accurate version number in the `package.json` file and read the version number from that.